<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaperCitationWindow</class>
 <widget class="QMainWindow" name="PaperCitationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>271</width>
      <height>102</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>168</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="button_copy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_go">
       <property name="text">
        <string>Go!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:#000000;
}

QPushButton:Pressed{
background-color:#24c8c8;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_copy">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>R
e
s
e
t</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinbox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>270</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="opaqueResize">
     <bool>false</bool>
    </property>
    <property name="handleWidth">
     <number>4</number>
    </property>
    <property name="childrenCollapsible">
     <bool>true</bool>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Input data for citation</string>
     </property>
     <widget class="QLabel" name="label_details">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>32</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Details</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_details">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>146</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_title">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>20</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_title">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>134</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_year">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_year">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>136</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_author">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>33</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_author">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>147</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radio_1st">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>One Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_2nd">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2nd Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_3rd">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3rd Author</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>315</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineedit_author</tabstop>
  <tabstop>lineedit_year</tabstop>
  <tabstop>lineedit_title</tabstop>
  <tabstop>lineedit_details</tabstop>
  <tabstop>button_go</tabstop>
  <tabstop>button_copy</tabstop>
  <tabstop>button_reset</tabstop>
  <tabstop>button_back</tabstop>
  <tabstop>radio_1st</tabstop>
  <tabstop>radio_2nd</tabstop>
  <tabstop>radio_3rd</tabstop>
  <tabstop>spinbox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
